### DISCUSSION TITLE
Spawnmob command

### DISCUSSION THREAD

**Discussion by Z1xus** [EssentialsX/Essentials#4127] [Ideas and Feature Suggestions] [answered]
/spawnmob doesn't have shulker_bullet entity, it's a little sad, could you add that in the next build?

**Comment by pop4959** [comment, marked as answer]
Shulker bullets aren't really a mob so much as they are a projectile, like arrows and snowballs, so I'm not sure how much it would make sense to add it to /spawnmob

**Reply by Z1xus** [reply]
Okay, thank you anyway
